Automattic/wp-calypso
# Jetpack Cloud: Refine Daily ActivityCards

#### Changes proposed in this Pull Request

* Refine `ActitivyCard`s in daily backup `ActivityCardList`
   * hide bottom action/toolbar when nothing is in it
   * hide actions for daily non-backup cards

![Untitled 9](https://user-images.githubusercontent.com/2810519/81238706-90d6ba00-8fb7-11ea-966b-b46205169bc0.png)

#### Testing instructions

1. Navigate to `/backups/activity` for a site with real-time backups
2. Verify it is unchanged vs `cloud.jetpack.com`
3. Navigate to  `/backups/activity` for a site with daily backups
4. Verify that:
   * The actions button only appears on full backups
   * when there is no items in in the bottom action/toolbar is hidden
